stockflux-securities-master/src/components/ToolTip.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./ToolTip.css";
+import PropTypes from "prop-types";
+
+const ToolTip = ({ children, message, className }) => {
+  return (
+    <div className={`tool-tip-container ${className ? className : ""}`}>
```

Comment:
Redundant `className` check.

Reply:
As className isn't being used in this component I have removed it as a prop after our discussion today